parser/preprocessed/SqlLexer_gen.cpp

```diff
@@ -1300,11 +1280,16 @@ extern int quickstep_yylex \
  */
 YY_DECL
 {
-	yy_state_type yy_current_state;
-	char *yy_cp, *yy_bp;
-	int yy_act;
+	register yy_state_type yy_current_state;
```

Comment:
Will 2.5.38+ produce a cpp file without the register type? Or I need to modify the file manually (even with a higher Flex version) by removing register keyword?